master/internal/api_tasks.go

```diff
@@ -247,14 +247,59 @@ func (a *apiServer) GetTaskAcceleratorData(
 	ctx context.Context,
 	req *apiv1.GetTaskAcceleratorDataRequest,
 ) (*apiv1.GetTaskAcceleratorDataResponse, error) {
-	return nil, grpcutil.UnimplementedError
+	if req.TaskId == "" {
+		return nil, status.Error(codes.InvalidArgument, "task ID missing")
+	}
+	res := []model.AcceleratorData{}
+	var accelerationData []*apiv1.AcceleratorData
+	err := db.Bun().NewSelect().
```

Comment:
what do we do in the case of someone queries for an task_id with one or more allocations that exist but do not have accelerator data (maybe they were created before this change or they were created but haven't sent the request to set the data yet)?

It doesn't look like this would error but maybe write a test for that case and make sure it doesn't error?

Reply:
Done